### PR TITLE
CLOUDP-208220 Add --decompress to atlas logs download command

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Add '--decompress' to `atlas logs download` command

_Jira ticket:_ CLOUDP-208220

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/atlas/deployments/logs_test.go [hunk 1]
```diff
@@ -85,45 +83,30 @@ func TestLogs_RunLocal(t *testing.T) {
...
 		downloadStore:  mockStore,
 		host:           "test",
 		name:           "mongodb.gz",
 	}
 
 	downloadOpts.Fs = afero.NewMemMapFs()
-	downloadOpts.Out = downloadOpts.name
 	deploymentTest.CommonAtlasMocks(downloadOpts.ProjectID)
 
+	b, _ := base64.RawStdEncoding.DecodeString("H4sIAAAAAAAA/8pIzcnJVyjPL8pJAQQAAP//hRFKDQsAAAA") // "hello world" gzipped
```
Comment (wtrocki) [inline internal/cli/atlas/deployments/logs_test.go:104]: Nice!

Comment (fmenezes) [reply internal/cli/atlas/deployments/logs_test.go:104]: yeap, that is to test the decompression code

**Review by wtrocki — APPROVED** [review]
LGTM. I'm not sure about size of our logs in production and how golang will handle some large zip files. I assume that we tested cli with +10mb gzip

**Comment by wtrocki** [comment]
NOTE. Pointing to empty gzip file (you have cluster but no logs) will fail. This is backend problem that should likely to fail rather than return empty gzip. I think we should handle this case by checking if gzip size

**Comment by fmenezes** [comment]
> LGTM. I'm not sure about size of our logs in production and how golang will handle some large zip files. I assume that we tested cli with +10mb gzip

My way of dealing with it is to stream 1kb each time straight to stdout, so no memory problems have manifested

**Comment by fmenezes** [comment]
> NOTE. Pointing to empty gzip file (you have cluster but no logs) will fail. This is backend problem that should likely to fail rather than return empty gzip. I think we should handle this case by checking if gzip size == 0

In the store we check for [empty response](https://github.com/mongodb/mongodb-atlas-cli/blob/1744e33397cfb92f248e537f25168c2ebceea88d/internal/store/logs.go#L89-L91) but I've also added the check if nothing is ever written

**Review by wtrocki — DISMISSED** [review]

File: docs/atlascli/command/atlas-logs-download.txt [hunk 2]
```diff
@@ -61,6 +61,10 @@ Options
      - Type
      - Required
      - Description
+   * - -d, --decompress
+     - 
+     - false
+     - Decompress log files
```
Comment (corryroot) [inline docs/atlascli/command/atlas-logs-download.txt:67]:
```suggestion
     - Flag that indicates whether to decompress the log files.
```

File: docs/mongocli/command/mongocli-atlas-logs-download.txt [hunk 3]
```diff
@@ -61,6 +61,10 @@ Options
      - Type
      - Required
      - Description
+   * - -d, --decompress
+     - 
+     - false
+     - Decompress log files
```
Comment (corryroot) [inline docs/mongocli/command/mongocli-atlas-logs-download.txt:67]:
```suggestion
     - Flag that indicates whether to decompress the log files.
```

**Review by corryroot — CHANGES_REQUESTED** [review]
Thanks! I commented with a minor copy suggestion.

**Review by corryroot — APPROVED** [review]
LGTM!

**Pull request merged**